src/main/java/core/basesyntax/figure/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figure;
+
+public class Square extends AbstractFigure implements Figure {
```

Comment:
The class 'Square' should not extend 'AbstractFigure' if 'AbstractFigure' is used to set behavior for classes. Consider using an interface to define behavior and use inheritance only for shared state or functionality.